python/grapher.go

```diff
@@ -274,3 +309,19 @@ type RawRef struct {
 	End       uint32
 	ToBuiltin bool
 }
+
+func installPipRequirements(pipBin string, requirementFiles []string) {
+	for _, requirementFile := range requirementFiles {
+		err := runCmdStderr(exec.Command(pipBin, "install", "-r", requirementFile))
+		if err != nil {
+			log.Printf("Error installing dependencies in %s. Trying piecemeal install", requirementFile)
+			if b, err := ioutil.ReadFile(requirementFile); err == nil {
+				for _, req := range strings.Split(string(b), "\n") {
+					runCmdLogError(exec.Command(pipBin, "install", req))
```

Comment:
As I wrote in the next comment, I didn't want to change the logic too much and this function is extracted from a code path which was ran in docker mode (preciously I had multiple calls for this function, but not in this version).

Which one do you think is best? Fail fast vs analyze 90%? It would be nice that we could analyze as much as possible but still communicate the warnings/errors back to the end user.